src/main/java/tr/ogu/edu/school/schoolapp/model/Term.java

```diff
@@ -36,11 +40,14 @@ public class Term {
 	@ManyToMany
 	@JoinTable(name = "student_term", joinColumns = @JoinColumn(name = "term_id"), inverseJoinColumns = @JoinColumn(name = "student_id"))
 	private Set<Student> students;
-
-	public Term(String termName, Date startDate, Date endDate, Set<Student> students) {
+	@OneToMany(mappedBy = "term")
```

Comment:
Daha önce konuştuğumuz yapıda term ile installement arasında one to many ilişkisi var ve payment ile installment arasında da yine one to many ilişkisi vardı. Sorguları buna uygun şekilde yapmamız gerekmiyor muydu? @ykartal

Reply:
Bu kısmı tartışalım,sanki veritabanını olması gerektiği gibi tasarlamamışız gibi geldi. Üçümüz de farklı düşünüyoruzbence. Ortaklaşmak için scrumda konuşalım.